<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="choix_algo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>79</y>
      <width>451</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <item>
     <property name="text">
      <string>SVC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Foret aléatoire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arbre de décision</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Réseau de neurones</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Algorithme utilisé :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Palladio L</family>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modèles de Machine Learning</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>175</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Données d'entrée :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="fichier_selec">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Browse_bouton">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Sortie : </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="sortie">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="launch_bouton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Lancer l'algorithme</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
